pkgs/tools/archivers/ctrtool/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "ctrtool";
-  version = "0.15";
+  version = "0.16";
 
   src = fetchFromGitHub {
-    owner  = "profi200";
+    owner  = "jakcron";
```

Comment:
Please explain why this change can be trusted, preferably citing an official source.